### DISCUSSION TITLE
error C2666: “fmt::v8::format_to”:

### DISCUSSION THREAD

**Discussion by kanli201904** [gabime/spdlog#3072] [Q&A]
using visualstudio 2015 to build: there was one error "error C2666: “fmt::v8::format_to”?    did you met the similar problem?

**Comment by tt4g** [comment]
Error codes alone do not determine the cause of the compile error.
Please provide a complete error message.

**Comment by kanli201904** [comment]
thanks, i tried the below version  that worked well.    
#define SPDLOG_VER_MAJOR 1
#define SPDLOG_VER_MINOR 8
#define SPDLOG_VER_PATCH 0

**Reply by tt4g** [reply]
What we need is not version information. Please provide the entire compilation error message and the source code that caused the error.
